### PR TITLE
Bug 1707096: gather: suffix container logs with container ids for uniqueness

### PR DESCRIPTION
**Pull request by deads2k**
related to Related to https://bugzilla.redhat.com/show_bug.cgi?id=1707096

### CODE REVIEW

File: data/data/bootstrap/files/usr/local/bin/installer-gather.sh [hunk 1]
```diff
@@ -14,14 +14,14 @@ mkdir -p "${ARTIFACTS}/bootstrap/containers"
...
-    sudo crictl inspect "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}.inspect"
+    sudo crictl logs "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}-${container}.log"
+    sudo crictl inspect "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}-${container}.inspect"
 done
 mkdir -p "${ARTIFACTS}/bootstrap/pods"
 sudo podman ps --all --quiet | while read -r container
 do
-    sudo podman logs "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.log"
-    sudo podman inspect "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.inspect"
+    sudo podman logs "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}-${container}.log"
```
Comment (sdodson) [inline data/data/bootstrap/files/usr/local/bin/installer-gather.sh]: This seems off.

Comment (wking) [reply data/data/bootstrap/files/usr/local/bin/installer-gather.sh]: /hold

Can we drop the `${container}-${container}` doubling here and in `installer-masters-gather.sh `?

**Comment by wking** [comment]
I changed "BUG" -> "Bug" in the PR subject to match [this regexp][1].

[1]: https://github.com/openshift/origin/blob/90b79ec8eee1c0eaf53b371fba69a6ceeeb0e2c4/pkg/oc/cli/admin/release/git.go#L155

**Pull request merged**